packages/lib/schemas/group.schema.json

```
@@ -0,0 +1,23 @@
+{
+  "$id": "group.schema.json",
```

Consider calling it something like this instead?

```suggestion
  "$id": "libAction.schema.json",
```